### ISSUE TITLE
[NPC] Skinning Keristraza in Nexus

### ISSUE THREAD

**Issue by Zelwindin** [dalaranwow/dalaran-wow#3748]
**Description**: The last boss in Nexus normal (have not tested in heroic) cannot be skinned. I have 450 skinning so this should not be an issue.

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Nexus normal, Keristraza, skinning. When you try, the mouse pointer changes to a "cast spell" icon, and clicking Keristraza again changes the mouse back to the normal behavior.

**Expected behaviour**: Tell us what should happen instead

Should be able to skin this boss with the appropriate level of skill.

**Steps to reproduce the problem**:
1. Kill the last boss in Nexus, Keristraza.
2. Attempt to Skin.
3. It will fail.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Keristrasza ID 26723
Skinning ID 393

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Serpentsting** [comment]
Same in heroic.

**Comment by sys6x** [comment]
Exactly as described, H or normal.  Also, if you want back to the normal cursor, hitting ESC is necessary.

**Comment by Rushor** [comment]
skinning_loot is assigned correctly in creature template hmm

**Comment by sys6x** [comment]
Is there an order of functions? Or a type of loot once the non-skinning
loot is picked up?  Or an if/else/then that'a mischecked?

**Comment by Rushor** [comment]
do you know if Keristraza has some kind of faction-loot? sth that cannot be looted by horde?

**Comment by sys6x** [comment]
Just noted exactly the same behavior on Cyanigosa in VH

**Comment by Sophilyplum** [comment]
@Rushor Keristraza should not have faction-specific loot.

**Comment by Rushor** [comment]
DB/Creature: Keristraza - Enable skinning

* closes https://github.com/dalaranwow/dalaran-wow/issues/3748
* commit: 96c84f2408aa96621f0dd1bb52b0394c1454dce5

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
